code/modules/surgery/operations/operation_zombie.dm

```diff
@@ -0,0 +1,74 @@
+/datum/surgery_operation/limb/bionecrosis
+	name = "induce bionecrosis"
+	rnd_name = "Bionecroplasty (Necrotic Revival)"
+	desc = "Inject reagents that stimulate the growth of a Romerol tumor inside the patient's brain."
+	rnd_desc = "An experimental procedure which induces the growth of a Romerol tumor inside the patient's brain."
+	implements = list(
+		/obj/item/reagent_containers/syringe = 1,
+		/obj/item/pen = 3.33,
+	)
+	time = 5 SECONDS
+	operation_flags = OPERATION_MORBID | OPERATION_LOCKED | OPERATION_NOTABLE
+	all_surgery_states_required = SURGERY_SKIN_OPEN|SURGERY_VESSELS_CLAMPED|SURGERY_BONE_SAWED
+	var/list/zombie_chems = list(
+		/datum/reagent/medicine/rezadone,
+		/datum/reagent/toxin/zombiepowder,
+	)
+
+/datum/surgery_operation/limb/bionecrosis/get_default_radial_image()
+	return get_dynamic_human_appearance(species_path = /datum/species/zombie)
+
+/datum/surgery_operation/limb/bionecrosis/all_required_strings()
+	. = ..()
+	. += "the limb must have a brain present"
```

Comment:
```suggestion
	. += "the head must have a brain present"
```

Reply:
With the existence of H.A.R.S. the brain is *technically* not guaranteed to be in the patient's head